### PR TITLE
RELATED: SD-933 version 7.5.0

### PR DESCRIPTION
**Pull request by naibinh**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/314/ : SUCCESS in 1m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/152/ : SUCCESS in 4s

File: website/versioned_docs/version-7.5.0/filter_visual_components.md [hunk 1]
```diff
@@ -0,0 +1,545 @@
...
+## Attribute filter
+
+All filters are defined using the `displayForm` identifiers. Attribute filters (both `positiveAttributeFilter` and `negativeAttributeFilter`) can be defined to match attribute elements by their URI (this is the default) or value (text filter).
+To use text filters, define the `textFilter` property of the filter and use values instead of URIs in the `in` or `notIn` arrays. 
+
+**NOTE:** Specifying attribute elements by value may slow down performance.
+
+### Positive attribute filter
+
+A **positive attribute filter** lists only those items whose attribute elements' URIs are included in the `in` property array.
```
Comment (ptquang86) [inline website/versioned_docs/version-7.5.0/filter_visual_components.md:26]: `elements'` => `elements's`?

File: website/versioned_docs/version-7.5.0/filter_visual_components.md [hunk 2]
```diff
@@ -0,0 +1,545 @@
...
+        },
+        in: ['<attribute-element-value-1>', '<attribute-element-value-2>'], // Attribute element values
+        textFilter: true
+    }
+},
+```
+
+### Negative attribute filter
+
+A **negative attribute filter** lists only those items whose attribute elements' URIs are *not* included in the `notIn` property array.
```
Comment (ptquang86) [inline website/versioned_docs/version-7.5.0/filter_visual_components.md:57]: `elements'` => `elements's`?

File: website/versioned_docs/version-7.5.0/time_over_time_comparison.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
+---
+title: Time Over Time Comparison
+sidebar_label: Time Over Time Comparison
+copyright: (C) 2007-2018 GoodData Corporation
```
Comment (ptquang86) [inline website/versioned_docs/version-7.5.0/time_over_time_comparison.md:4]: `2020`?

File: website/versioned_docs/version-7.5.0/pivot_table_component.md [hunk 4]
```diff
@@ -0,0 +1,280 @@
+---
+title: Pivot Table
+sidebar_label: Pivot Table
+copyright: (C) 2007-2018 GoodData Corporation
```
Comment (ptquang86) [inline website/versioned_docs/version-7.5.0/pivot_table_component.md:4]: `2020`?

File: website/versioned_docs/version-7.5.0/filter_visual_components.md (hunk 2 above)
Comment (naibinh) [reply website/versioned_docs/version-7.5.0/filter_visual_components.md:57]: Please check [Rule 2.a](https://www.grammarbook.com/punctuation/apostro.asp) out

File: website/versioned_docs/version-7.5.0/filter_visual_components.md (hunk 1 above)
Comment (naibinh) [reply website/versioned_docs/version-7.5.0/filter_visual_components.md:26]: Please check [Rule 2.a](https://www.grammarbook.com/punctuation/apostro.asp) out

File: website/i18n/en.json [hunk 5]
```diff
@@ -1211,6 +1211,34 @@
       "version-7.4.0/version-7.4.0-pivot_table_component": {
         "title": "Pivot Table",
         "sidebar_label": "Pivot Table"
+      },
+      "version-7.5.0/version-7.5.0-attribute_element_option": {
```
[the remark this answers is not in the document]
Comment (naibinh) [reply website/i18n/en.json:1215]: There is only `attribute_element_option.md` file so it's correct

File: website/versioned_docs/version-7.5.0/filter_visual_components.md [hunk 6]
```diff
@@ -0,0 +1,545 @@
...
+title: Filter Visual Components
+sidebar_label: Filter Visual Components
+copyright: (C) 2007-2020 GoodData Corporation
+id: version-7.5.0-filter_visual_components
+original_id: filter_visual_components
+---
+
+This article provides examples of filtering visual components by attribute, date, and measure values.
+
+You can filter the `Visualization` component, `Kpi`, `Headline`, chart components, and table components with the `filters` prop. The `filters` prop is an array of attribute filters and date filters. You can make the filters dynamic with the [`AttributeFilter`](attribute_filter_component.md), [`AttributeElements`](create_custom_attribute_filter.md), [`DateFilter`](date_filter_component.md), and [`MeasureValueFilterDropdown`](measure_value_filter_component.md) components.
```
[the remark this answers is not in the document]
Comment (naibinh) [reply website/versioned_docs/version-7.5.0/filter_visual_components.md:11]: Honestly, I missed this guide line for Geo chart at the time I made `geo_pushpin_chart_component.md`.
But I think `...chart components...` phrase is including our Geo chart.

File: website/versioned_docs/version-7.5.0/time_over_time_comparison.md (hunk 3 above)
Comment (naibinh) [reply website/versioned_docs/version-7.5.0/time_over_time_comparison.md]: Fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/322/ : SUCCESS in 34s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/159/ : SUCCESS in 3s

File: website/versioned_docs/version-7.5.0/pivot_table_component.md (hunk 4 above)
Comment (naibinh) [reply website/versioned_docs/version-7.5.0/pivot_table_component.md]: Fixed

File: website/versioned_docs/version-7.5.0/create_custom_attribute_filter.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
+---
+id: version-7.5.0-create_custom_attribute_filter
+title: Create a Custom Attribute Filter
+sidebar_label: Create Custom Filter
+copyright: (C) 2007-2018 GoodData Corporation
```
[the remark this answers is not in the document]
Comment (naibinh) [reply website/versioned_docs/version-7.5.0/create_custom_attribute_filter.md]: Fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/323/ : SUCCESS in 1m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/160/ : SUCCESS in 3s

File: website/versioned_docs/version-7.5.0/filter_visual_components.md (hunk 1 above)
Comment (ptquang86) [reply website/versioned_docs/version-7.5.0/filter_visual_components.md:26]: Thanks :)

**Comment by ptquang86** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/324/ : SUCCESS in 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/161/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/325/ : SUCCESS in 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/91/ : SUCCESS in 12s

**Pull request merged**